app/views/units/_form.html.slim

```diff
@@ -22,9 +22,9 @@
   = f.input :ends_at, wrapper: :inline_date, start_year: 2021, end_year: 2021
```

Comment:
how is the exact work flow of editing guest troop-amount etc?

- the admin sets the flag :guest_troop?
- the AL/LL sets the expected number with this form? because the lines are missing. 

then we should add the files to the form. in the same step we should not show the form-fields of the imported values to the user, because they would be overwritten:

```rb
  .row
    - if can?(:manage, @unit) 
      .col-6
	    = f.input :expected_participants_f
	    = f.input :expected_participants_m
	  .col-6
	    = f.input :expected_participants_leitung_f
	    = f.input :expected_participants_leitung_m
    .col-6
      = f.input :expected_guest_participants
      = f.input :expected_guest_leaders
```

the import will not harm this field, resp. not change it. but it will just be empty. is it planned that the midata-export will provide those fields?

Reply:
the guests come frome a one time import from an excel. they don't have to edit the fields, everything comes via database